modules/mtl/arrow-mtl-data/src/test/kotlin/arrow/mtl/EitherTTest.kt

```diff
@@ -62,17 +59,23 @@ class EitherTTest : UnitSpec() {
         constEQK
       ),
 
-      AlternativeLaws.laws(
-        EitherT.alternative(Id.monad(), Int.monoid()),
-        { EitherT.just(Id.applicative(), it) },
-        { i -> EitherT.just(Id.applicative(), { j: Int -> i + j }) },
-        idEQK
-      ),
+      /*
+      TODO:
+       Alternative laws right distributivity is not implemented correctly
+       https://github.com/arrow-kt/arrow/issues/1880
+
+       AlternativeLaws.laws(
+         EitherT.alternative(Id.monad(), Int.monoid()),
+         EitherT.genK(Id.genK(), Gen.int()),
+         idEQK
+       ),
+       */
```

Comment:
Please run the alternative test and just comment out right distributivity and make the note there as this time around it's not actually the datatype that is broken :)

Reply:
ok, got it